wled00/FX_fcn.cpp

```diff
@@ -692,6 +716,9 @@ void Segment::setPalette(uint8_t pal) {
   if (pal < 245 && pal > GRADIENT_PALETTE_COUNT+13) pal = 0; // built in palettes
   if (pal > 245 && (strip.customPalettes.size() == 0 || 255U-pal > strip.customPalettes.size()-1)) pal = 0; // custom palettes
   if (pal != palette) {
+#ifndef WLED_DISABLE_MODE_BLEND
```

Comment:
Not sure I like this. Palettes have their own option for blending and this introduces inconsistency.

Reply:
The original palette blending still works and takes precedence over this if enabled.

Reply:
If `strip.paletteFade` is `false` and `modeBlending` is `true`, the palette will still be blended. This is not a desired outcome.

Reply:
That depends on the transition style. Yes, we start a transition if `strip.paletteFade` is `false` and `modeBlending` is `true`, but `currentPalette` was modified so that the palette is not blended, instead just like with `currentBri` and `currentColor` we get the transitional palette if `_modeBlend` is true. With the Fade style, this looks similar to palette blending. But with all the other styles the looks are very different.

Before I added this change, the transitions didn't take palette changes into account and looked quite odd.